rtnetlink/src/link/get.rs

```diff
@@ -34,6 +34,17 @@ impl LinkGetRequest {
         }
     }
 
+    /// Setting filter mask(e.g. RTEXT_FILTER_BRVLAN and etc)
+    pub fn set_filter_mask(mut self, family: u8, filt_mask: u32) -> Self {
```

Comment:
This is kind a bit redundant with `new()`, could we make this simply:

```rust
pub fn set_filter_mask(mut self, family: u8, filt_mask: u32) -> Self {
     self.message.header.interface_family = family;
     self.message.nlas.push(Nla::ExtMask(filter_mask));
     self
}
```

Then it would be called like:

```rust
LinkGetRequest::new(handle)
    .set_filter_mask(family, filt_mask)
    .execute()
```

Also, do we necessarily need the address family when setting a filter?
Or should this be a different `set_interface_family()` method?

Reply:
Changing the `LinkGetRequest::new()` would break the API which I think we can discuss that in another pull request while this PR focusing on extending existing API for `filter mask`.
But if you prefer to do it in a bundle, I am OK with it, just let me know.

Reply:
I tested several days ago, it is required for vlan filtering, but let me test again.

Reply:
It it required to set `interface_family`. Without it, the `AfBridgeSpec` will not show. Tested on kernel 5.4.43(archlinux).

For kernel code, the `validate_linkmsg()` in `net/core/rtnetlink.c` will ensure the returned AfSpec interface family matches, and `br_fill_ifinfo()` in `net/bridge/br_netlink.c` set the `hdr->ifi_family = AF_BRIDGE`.

Reply:
Sounds good, thank you.